From f41bdfb0540ebeff758370da7688bc0e6ddc90c4 Mon Sep 17 00:00:00 2001
From: wouterlucas <wouter@wouterlucas.com>
Date: Fri, 18 Jan 2019 13:42:09 -0800
Subject: [PATCH] [IRRemote] Do no use static directories in includes, lets
 handle that in cmake as different build environments have different
 subdirectories for refsw

---
 RemoteControl/IRRemote.cpp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/RemoteControl/IRRemote.cpp b/RemoteControl/IRRemote.cpp
index b33ca82..b5df2d1 100644
--- a/RemoteControl/IRRemote.cpp
+++ b/RemoteControl/IRRemote.cpp
@@ -3,10 +3,10 @@
 
 #include <linux/uinput.h>
 #include <interfaces/IKeyHandler.h>
-#include <refsw/nexus_config.h>
-#include <refsw/nexus_platform.h>
-#include <refsw/nxclient.h>
-#include <refsw/nexus_input_client.h>
+#include <nexus_config.h>
+#include <nexus_platform.h>
+#include <nxclient.h>
+#include <nexus_input_client.h>
 
 extern int getIRHandle();
 
-- 
2.11.0

